flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/table/JdbcDynamicTableSourceITCase.java

```diff
@@ -276,9 +280,88 @@ public void testFilter() {
     @ParameterizedTest
     @EnumSource(Caching.class)
     void testLookupJoin(Caching caching) {
+        Collection<Row> dataToRegister =
+                Arrays.asList(
+                        Row.of(1L, "Alice"),
+                        Row.of(1L, "Alice"),
+                        Row.of(2L, "Bob"),
+                        Row.of(3L, "Charlie"));
+
+        String createTableStatement =
+                "CREATE TABLE value_source ( "
+                        + " `id` BIGINT, "
+                        + " `name` STRING, "
+                        + " `proctime` AS PROCTIME()"
+                        + ") WITH ("
+                        + " 'connector' = 'values', "
+                        + " 'data-id' = '%s'"
+                        + ")";
+        String selectStatement =
+                "SELECT S.id, S.name, D.id, D.timestamp6_col, D.decimal_col FROM value_source"
+                        + " AS S JOIN jdbc_lookup for system_time as of S.proctime AS D ON S.id = D.id";
```

Comment:
This seems to not exercise the filter pushdown code? Can you change it such that it is being exercised?

Reply:
This was a test without the filter. The filter tests are later in this file see line 425 for an example.